tests/testthat/test-est.incidence.R

```diff
@@ -79,3 +79,36 @@ test_that("`est.incidence()` produces expected results", {
 
   expect_equal(est_true, est_false)
 })
+
+test_that("est.incidence() aborts when strata is missing in `pop_data`", {
+  # get pop data
+  xs_data <- load_pop_data(
+    file_path = "https://osf.io/download//n6cp3/",
+    age = "Age",
+    value = "result",
+    id = "index_id",
+    standardize = TRUE
+  ) %>%
+    filter(Country == "Pakistan") %>%
+    slice_head(n = 100)
+
+  # get noise data
+  noise <- load_noise_params("https://osf.io/download//hqy4v/") %>%
+    filter(Country == "Pakistan")
+
+  # get curve data
+  curve <- load_curve_params("https://osf.io/download/rtw5k/")
+
+  expect_error(
+    object = est.incidence.by(
```

Comment:
why are we testing `est.incidence.by()` in the test file for `est.incidence()`? Please be careful to keep these tests properly organized.

Reply:
please add a test in which `strata` contains multiple elements that don't exactly match the columns of `pop_data`

Reply:
I have separated `est.incidence` and `est.incidence.by` into different scripts. In addition, I have added a test for multiple `strata` in `est.incidence.by`.